Resources/Public/JavaScript/Crud.js

```diff
@@ -121,11 +123,15 @@ define(['jquery', 'TYPO3/CMS/FrontendEditing/FrontendEditing'], function ($, Fro
 							F.trigger(F.REQUEST_COMPLETE);
 							F.trigger(F.CONTENT_CHANGE);
 						}
+
+						F.hideLoadingScreen();
```

Comment:
Why not move this outside both the if and else scope?

Reply:
The first hideLoadingScreen() call is async. The second is a catcher for any item that doesn't become an async request.